### PR TITLE
[vcpkg doc] Add Vietnamese Translation

### PR DESCRIPTION
**Pull request by sakiodre**
In order to help increase the community, I've added the README_vn.md translation.

Also fixed a typo in [README.md#security](https://github.com/microsoft/vcpkg#security) `in in` to `it in`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -321,7 +322,7 @@ places the associated license(s) in the location `installed/<triplet>/share/<por
 Most ports in vcpkg build the libraries in question using the original build system preferred
 by the original developers of those libraries, and download source code and build tools from their
 official distribution locations. For use behind a firewall, the specific access needed will depend
-on which ports are being installed. If you must install in in an "air gapped" environment, consider
+on which ports are being installed. If you must install in an "air gapped" environment, consider
```
Comment (Cheney-W) [inline README.md:325]:
```suggestion
on which ports are being installed. If you must install it in an "air gapped" environment, consider
```

**Review by Cheney-W — CHANGES_REQUESTED** [review]

**Comment by Cheney-W** [comment]
Moving the PR to draft. Please set it to "ready for review" once changes have been applied.

**Review by Cheney-W — APPROVED** [review]

**Comment by BillyONeal** [comment]
Dropping "reviewed" and adding docs folks.

We accepted the other readme translations fairly "blindly" but my understanding is that we got told not to do that by the real localizers involved when we really localized. But I'm not positive on the situation; hopefully one of @ras0219-msft @vicroms @AugP know how to proceed.

**Comment by sakiodre** [comment]
I've thought about that issue as well, I suggest comparing the diff file between this and README.md, comparing this against google translate results, and checking for black-listed words (one could be found [here](https://github.com/blue-eyes-vn/vietnamese-offensive-words/blob/main/vn_offensive_words.txt))

**Comment by BillyONeal** [comment]
I asked some localization folks and will keep you posted. Thanks for your contribution to vcpkg!

**Comment by Cheney-W** [comment]
Temporarily mark this PR as a draft.

**Comment by PhoebeHui** [comment]
Can someone understand Vienamese help review this PR?

**Pull request merged**